### PR TITLE
Setup DbSet in AppDbContext

### PR DESCRIPTION
**Pull request by rinesakuci**

### CODE REVIEW

File: pentasharp/Data/AppDbContext.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    public class AppDbContext : DbContext
+    {
+        public AppDbContext(DbContextOptions<AppDbContext> options) : base(options)
+        {
+        }
+
+        
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+
```
Comment (GentaSL) [inline pentasharp/Data/AppDbContext.cs]: remove extra lines

File: pentasharp/Data/AppDbContext.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+using Microsoft.EntityFrameworkCore;
+
+namespace pentasharp.Data 
+{
+    public class AppDbContext : DbContext
+    {
+        public AppDbContext(DbContextOptions<AppDbContext> options) : base(options)
+        {
+        }
+
```
Comment (GentaSL) [inline pentasharp/Data/AppDbContext.cs:9]: remove extra lines

File: pentasharp/Program.cs [hunk 3]
```diff
@@ -9,6 +12,9 @@ public static void Main(string[] args)
             // Add services to the container.
```
Comment (GentaSL) [inline pentasharp/Program.cs]: remove all comments in this file please, consider it as a clean-up of the class that was not created by you initially

File: pentasharp/appsettings.json [hunk 4]
```diff
@@ -5,5 +5,10 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "DefaultDbConnection": "Server=(localdb)\\MSSQLLocalDB;Database=PentasharpDB;Trusted_Connection=True;MultipleActiveResultSets=true"
```
Comment (GentaSL) [inline pentasharp/appsettings.json]: in Program.cs you are using DefaultConenction
    builder.Services.AddDbContext<AppDbContext>(options =>
                options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection")));
 , here you are referring as DeafultDBConnection. The name should match

File: pentasharp/appsettings.json [hunk 5]
```diff
@@ -5,5 +5,10 @@
...
+  }
+
```
Comment (GentaSL) [inline pentasharp/appsettings.json]: remove extra lines

File: pentasharp/Data/AppDbContext.cs (hunk 2 above)
Comment (GentaSL) [reply pentasharp/Data/AppDbContext.cs:9]: you should leave one line as space between AppDbContext, not more than one or less

File: pentasharp/Data/AppDbContext.cs [hunk 6]
```diff
@@ -0,0 +1,15 @@
+using Microsoft.EntityFrameworkCore;
+
+namespace pentasharp.Data 
+{
+    public class AppDbContext : DbContext
+    {
+        public AppDbContext(DbContextOptions<AppDbContext> options) : base(options)
+        {
+        }
```
Comment (GentaSL) [inline pentasharp/Data/AppDbContext.cs:10]: leave one extra line in-between methods for readability

**Review by oltahasimja — APPROVED** [review]

**Review by GentaSL — APPROVED** [review]

**Pull request merged**